6-anthos-install/README.md

```diff
@@ -0,0 +1,353 @@
+# Deploying Bank of Anthos 
+These instructions need to be run on the bastion host. To access the bastion host, open the Google Cloud console, and navigate to Compute Engine in the GKE project. In my instance, `prj-bu1-d-boa-gke` Then, select the `gce-bastion-us-west1-b-01` and click on `ssh`. You need to be a whitelisted member to access this node.
+
+
+## Insall required tools
+```console
+sudo su
+yum install git
+yum install google-cloud-sdk-kpt
+yum install jq -y
+sudo yum install kubectl
+exit
+```
+
+## Pull the repo
+```console
+cd ${HOME}
+git clone https://github.com/GoogleCloudPlatform/terraform-example-foundation-app.git
+```
+
+## Define required environment variables
+When indicated, make sure to replace the values below with the appropriate values based on the outcome of terraform.
+```console
+# replace YOUR_PROJECT_ID with the project id for the project that will host your clusters.
+# For example: prj-bu1-d-boa-gke-ecb0
+export PROJECT_ID=YOUR_PROJECT_ID
+export PROJECT_NUM=$(gcloud projects describe ${PROJECT_ID} --format='value(projectNumber)')
+export CLUSTER_1=gke-boa-us-east1-001
+export CLUSTER_1_REGION=us-east1
+export CLUSTER_2=gke-boa-us-west1-001
+export CLUSTER_2_REGION=us-west1
+export CLUSTER_INGRESS=gke-mci-us-east1-001
+export CLUSTER_INGRESS_REGION=us-east1
+export WORKLOAD_POOL=${PROJECT_ID}.svc.id.goog
+export MESH_ID="proj-${PROJECT_NUM}"
+export ASM_VERSION=1.8
+export ISTIO_VERSION=1.8.3-asm.2
+export ASM_LABEL=asm-183-2
+export CTX_1=$PROJECT_ID_$CLUSTER_1_REGION_CLUSTER_1
```

Comment:
should this be
```suggestion
export CTX_1="${PROJECT_ID}_$PCLUSTER_1}_${CLUSTER_1_REGION}"
```

Reply:
It should be Project ID, then region, then cluster name, so it's correct